scripts/capture.js

```diff
@@ -22,18 +18,21 @@ async function request(url) {
     return new Promise((resolve, reject) => {
         const req = https.request(url, options, (resp) => {
             let data = "";
-
             resp.on("data", (chunk) => {
                 data += chunk;
             });
-
             resp.on("end", () => {
+				// reject on bad status
+				if (resp.statusCode != 200) {
+				  console.log('url: ' + url);	
+                  return reject(new Error('statusCode=' + resp.statusCode + ' for URL: ' + url));
+                }
                 try {
                     resolve(JSON.parse(data));
                 }
-                catch (e) {
+                catch (e) {;
```

Comment:
Do you mind removing the unnecessary `;` sign here? Also, it would be great if you could use `Shift+Alt+F` to apply formatting on the document.